community/kernel/src/main/java/org/neo4j/kernel/impl/api/ConstraintEnforcingEntityOperations.java

```diff
@@ -67,10 +70,10 @@ public ConstraintEnforcingEntityOperations(
     public boolean nodeAddLabel( KernelStatement state, long nodeId, int labelId )
             throws EntityNotFoundException, ConstraintValidationKernelException
     {
-        Iterator<UniquenessConstraint> constraints = schemaReadOperations.constraintsGetForLabel( state, labelId );
+        Iterator<PropertyConstraint> constraints = uniquePropertyConstraints( schemaReadOperations.constraintsGetForLabel( state, labelId ) );
```

Comment:
Would it be beneficial to have separate methods to list different types of constraints on `SchemaReadOperations`?
Like: `SchemaReadOperations#uniquenessConstraintsGetForLabel` and `SchemaReadOperations#notNullConstraintsGetForLabel`?